src/ansys/grpc/dpf/_version.py

```diff
@@ -1,2 +1,2 @@
 """ansys-grpc-dpf python protocol version"""
-__version__ = '0.8.2dev0'  # major.minor.patch
+__version__ = '0.9.0dev0'  # major.minor.patch
```

Comment:
```suggestion
__version__ = '0.8.2dev0'  # major.minor.patch
```
This will be modified automatically by our pipelines upon switch to a 242 server